packages/markdown-docx/src/CiceroMarkToOOXML/rules.js

```diff
@@ -47,6 +47,24 @@ const EMPHASIS_RULE = (value) => {
     `;
 };
 
+/**
+ * Inserts strong text.
+ *
+ * @param {string}  value  Enclosing value of the OOXML tag
+ * @returns {string} OOXML tag for the strong text
+ */
+const STRONG_RULE = value => {
+    return `
+      <w:r>
+        <w:rPr>
+          <w:b />
+          <w:bCs />
```

Comment:
What is this tag for? `<w:b>` is already there. Do we need <w:bCs>` too?

Reply:
`bCs` is used for complex [script](https://c-rex.net/projects/samples/ooxml/e1/Part4/OOXML_P4_DOCX_bCs_topic_ID0EGNFO.html). 
This is the reason why I have used it.

Reply:
Are we using complex scripts? All templates are in English only, right?

Reply:
As far as i remember,  one of the templates used some different language. The template was probably `car rental`. Besides, it might be possible that later templates might be created in different languages. So i guess there would be no harm keeping this.

Reply:
> Besides, it might be possible that later templates might be created in different languages.

Good point. In that case, we can keep it.